include/ethernet.h

```diff
@@ -0,0 +1,92 @@
+/*
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *
+ *  Copyright (C) 2021  The DOSBox Team
```

Comment:
You wrote `ethernet.{h,cpp}` files from scratch - if you want to, you can add use your name and email in the copyright line (we do not require copyright assignment) :)

If you prefer to assign copyright to us - that's ok, please use "The DOSBox Staging Team"

*We are DOSBox Staging Team :) DOSBox Team is group that's maintaining old DOSBox code in SVN - we do not attribute new work to that team unless they accept/merge that code themselves to svn trunk.*